src/markdown.js

```diff
@@ -93,24 +110,48 @@ export const reportToComment = ({
     const nameToLink = (name) =>
         `https://github.com/${repository}/blob/${commit}/${name}`;
 
-    const fileTable = tableToMD({
-        headers: ['Path', 'FRE', 'GF', 'SMOG', 'ARI', 'CLI', 'LWF', 'DCRS'],
+    const readabilityTable = tableToMD({
+        headers: ['Path', 'Readability'],
+        rows: report.fileResults.map((result) => 
+            resultToReadabilityRowWithDiff(result, nameToLink),
+        ),
+    });
+
+    const detailedFileTable = tableToMD({
+        headers: ['Path', 'Readability', 'FRE', 'GF', 'ARI', 'CLI', 'DCRS'],
         rows: report.fileResults.flatMap((result) => [
             resultToScoreTableRow(result, nameToLink),
             resultToDiffTableRow(result),
         ]),
     });
 
     const averageTable = tableToMD({
-        headers: ['&nbsp;', 'FRE', 'GF', 'SMOG', 'ARI', 'CLI', 'LWF', 'DCRS'],
+        headers: ['&nbsp;', 'Readability', 'FRE', 'GF', 'ARI', 'CLI', 'DCRS'],
         rows: [
             resultToScoreTableRow(report.averageResult[0]),
             resultToDiffTableRow(report.averageResult[0]),
         ],
     });
 
+    const averageReadability = roundValue(report.averageResult[0].scores.readabilityScore);
+    const averageReadabilityDiff = addPositiveDiffMarker(roundValue(report.averageResult[0].diff.readabilityScore));
+
     return `
-Readability after merging this PR:
+Readability after merging this PR: ${averageReadability}/100 (${averageReadabilityDiff})
```

Comment:
suggestion: I think this may help make it clear that it's the overall figure 
(took me a split second)
```suggestion
Overall readability after merging this PR: ${averageReadability}/100 (${averageReadabilityDiff})
```